packages/experimental-loader/test/projects/errors-integration/webpack.config.ts

```diff
@@ -5,6 +5,7 @@ export default {
     entry: './index.js',
     context: __dirname,
     devtool: false,
+    output: { publicPath: '' },
```

Comment:
why is this needed? is wepack's "auto" not working?

Reply:
no will comment on this